public/js/utils/sources-tree.js

```diff
@@ -4,11 +4,6 @@ const { isPretty } = require("../utils/source");
 
 const IGNORED_URLS = ["debugger eval code", "XStringBundle"];
 
-function isHiddenSource(source) {
-  const url = source.get("url");
-  return !url || url.match(/SOURCE/) || IGNORED_URLS.includes(url);
-}
```

Comment:
It was only used in one place outside of this module, which was if it should be in the search list results. What you mean by "show a source tab"? We definitely should show a tab for any source that's displayed in the editor, even if it's an eval source (and that's what we do).

The rest of the app should work with any source, whether it's an eval source/IGNORE_URLS/etc source. Should always show up in the call stack, etc. The only consideration some components need to make it if it has a `null` URL or not, so that's all they need to check. I'm trying to be conservative on the amount of terms we use through the codebase, because "hidden" isn't very descriptive. An eval source is not "hidden", it's just not shown in the source tree. But you can definitely interact with it everywhere else.